recipes/openfst/all/conanfile.py

```diff
@@ -0,0 +1,212 @@
+
+    @functools.lru_cache(1)
+    def _configure_autotools(self):
+        autotools = AutoToolsBuildEnvironment(self)
+        yes_no = lambda v: "yes" if v else "no"
+        args = [
+            "--with-pic={}".format(yes_no(self.options.get_safe("fPIC", True))),
+            "--enable-shared={}".format(yes_no(self.options.shared)),
+            "--enable-static={}".format(yes_no(not self.options.shared)),
+            "--enable-bin={}".format(yes_no(self.options.enable_bin)),
+            "--enable-compact-fsts={}".format(yes_no(self.options.enable_compact_fsts)),
+            "--enable-compress={}".format(yes_no(self.options.enable_compress)),
+            "--enable-const-fsts={}".format(yes_no(self.options.enable_const_fsts)),
+            "--enable-far={}".format(yes_no(self.options.enable_far)),
+            "--enable-grm={}".format(yes_no(self.options.enable_grm)),
+            "--enable-linear-fsts={}".format(yes_no(self.options.enable_linear_fsts)),
+            "--enable-lookahead-fsts={}".format(yes_no(self.options.enable_lookahead_fsts)),
+            "--enable-mpdt={}".format(yes_no(self.options.enable_mpdt)),
+            "--enable-ngram-fsts={}".format(yes_no(self.options.enable_ngram_fsts)),
+            "--enable-pdt={}".format(yes_no(self.options.enable_pdt)),
+            "--enable-special={}".format(yes_no(self.options.enable_special)),
+            "LIBS=-lpthread",
+        ]
+        if tools.get_env("CC"):
+            args.append("CC={}".format(tools.get_env("CC")))
+        if tools.get_env("CXX"):
+            args.append("CXX={}".format(tools.get_env("CXX")))
+        autotools.configure(args=args, configure_dir=self._source_subfolder)
+        return autotools
+
+    def export_sources(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            self.copy(patch["patch_file"])
+
+    def _patch_sources(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+
+    def build(self):
+        self._patch_sources()
+        autotools = self._configure_autotools()
+        autotools.make()
+
+    def package(self):
+        self.copy(pattern="COPYING", dst="licenses", src=self._source_subfolder)
+        autotools = self._configure_autotools()
+        autotools.install()
+
+        lib_dir = os.path.join(self.package_folder, "lib")
+        lib_subdir = os.path.join(self.package_folder, "lib", "fst")
+        if os.path.exists(lib_subdir):
+            for fn in os.listdir(lib_subdir):
+                tools.rename(os.path.join(lib_subdir, fn), os.path.join(lib_dir, "lib{}".format(fn)))
+            tools.rmdir(lib_subdir)
+
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+        tools.remove_files_by_mask(lib_dir, "*.la")
+
+    @property
+    def _get_const_fsts_libs(self):
+        libs = []
+        for n in [8, 16, 64]:
+            libs.append("const{}-fst".format(n))
+        return libs
```

Comment:
using [list comprehansions](https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions) it coulld be written as one-liiner, e.g.:
`return ["const{}-fst".format(n) for n in [8, 16, 64]]`
no need to change, just FYI